Core/Inc/main.h

```diff
@@ -37,6 +37,15 @@ extern "C" {
 
 /* Exported types ------------------------------------------------------------*/
 /* USER CODE BEGIN ET */
+CAN_HandleTypeDef hcan1;
```

Comment:
Should leave these in the main.c file and extern them here.

Reply:
Fixed in latest commit